README.md

```diff
@@ -1,7 +1,214 @@
-#TBA...
+# ApiProvider
 
-## Features and bugs
+### Clean and boilerplate free network requests with only a few lines of code.
```

Comment:
should we mention that we use `https://pub.dev/packages/http` ?

Reply:
Probably yes. You could write something like: `based on dart:http`.

Reply:
went for Build around [dart:http](https://pub.dev/packages/http).